src/client/api.ts

```diff
@@ -180,6 +187,39 @@ const getDepositorLPAssociatedTokenAddress = multiAsync(
 	}
 );
 
+const getGatewayToken = multiAsync(
+	async (connection: Connection, wallet: Wallet, userPK: PublicKey) => {
+		const GATEWAY_TOKEN_ACCOUNT_OWNER_FIELD_OFFSET = 2;
```

Comment:
just a question: where does this come from?

Reply:
I copied this from the native method of `findGatewayToken` to be able to insert our own gateway programId